db/migrate/20200921163903_create_users.rb

```diff
@@ -1,8 +1,7 @@
 class CreateUsers < ActiveRecord::Migration[6.0]
   def change
     create_table :users do |t|
```

Comment:
Never update a migration, always create a new one

Reply:
this can cause big issues

Reply:
OK, understood